File: learning-python/leet-code/src/remove_element/remove_element.py
def removeElement(nums, val):
	i = 0
	while i < len(nums):
		# check if element is the val
		if nums[i] == val:
			nums.remove(val)
			continue
		i += 1
		# if it is remove it then push an underscore. increment count
	return len(nums) 

File: learning-javascript/leet-code/src/remove-element/remove-element.ts
export function removeElement(nums: number[], val: number): number {
	let i = 0;
	while(i<nums.length) {
		// check if element is the val
		if (nums[i] === val) {
			nums.splice(i,1);
			continue;
		}
		i++;
		// if it is remove it then push an underscore. increment count
	}
	return nums.length;
}